include/odp/api/spec/classification.h

```diff
@@ -55,6 +55,14 @@ extern "C" {
  * @deprecated Use ODP_PMR_INVALID instead
  */
 
+/**
+ * Invalid mark value.
+ * This value returned from odp_packet_mark() when the mark
+ * is not valid for the packet or mark support is not available.
+ * @see odp_cls_capability_t::max_mark
+ */
+#define ODP_CLS_MARK_INVALID 0
```

Comment:
Usually, invalid values are needed with handles. I think we don't need explicit invalid value for the mark, but just define that the default value for mark is 0.

Reply:
I will change to ODP_CLS_MARK_DEFAULT in next version.